app/javascript/src/components/Dashboard/Contacts/DeleteAlert.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { Alert, Toastr } from "neetoui";
+
+const DeleteAlert = ({ onClose, setShowDeleteAlert }) => {
+  const deleteNotePrompt = "Delete this Contact?";
+  const deleteNoteMessage =
+    "Are you sure you want to continue? This cannot be undone.";
+  const toasterSuccess = "Contact deleted successfully";
```

Comment:
These variables are pretty unnecessary at this moment within this component. Even if these variables were not present the idea would have been conveyed in this component because these are straightforward messages. The component is very small and these variables don't have reusability.

But I would like to know your reasoning for adding these variables. Maybe you had seen a use case for them which I am not able to see, yet.

Reply:
I added the messages as variables as I felt it was better to just use the variable in-place of the actual string.
But as you had mentioned, the strings don't have re-usability, and the scope of usage for the component is just for the contact namespace, I think it is better to just use them as direct string messages than adding them as a variable.